Author: Reiner Herrmann <reiner@reiner-h.de>
Description: Some documentation fixes
 - add @dircategory / @direntry (found by Lintian)
 - fix wrong section level
 - use makeinfo instead of obsolete texi2html

--- a/docs/makefile
+++ b/docs/makefile
@@ -23,9 +23,7 @@
 
 html/usermanual_1.html: usermanual.texi
 	for d in html; do [ -d $$d ] || mkdir -p $$d || exit 1; done
-	texi2html --expand tex --split chapter --no-iftex usermanual.texi
-	mv usermanual/*.html html
-	rmdir usermanual
+	makeinfo --html --no-split --no-headers -o usermanual.html usermanual.texi
 
 ship:
 	-rm -f *.{aux,cp,cps,dvi,fn,ky,log,pg,toc,tp,vr}
--- a/docs/usermanual.texi
+++ b/docs/usermanual.texi
@@ -5,6 +5,11 @@
 @setchapternewpage odd
 @c %**end of header
 
+@dircategory Education
+@direntry
+* UCBLogo: (ucblogo).	Logo Programming Language
+@end direntry
+
 @iftex
 @vsize = 8.9 in
 @parskip = 7 pt
@@ -2875,7 +2880,7 @@
 
 
 @node  AND, OR, LOGICAL OPERATIONS, LOGICAL OPERATIONS
-@unnumberedsubsec and
+@unnumberedsec and
 @cindex and
 
 @example
@@ -6766,7 +6771,7 @@
 @end menu
 
 @node dMACRO, dDEFMACRO, MACROS, MACROS
-@unnumberedsubsec .macro
+@unnumberedsec .macro
 @cindex .macro
 @cindex .defmacro
 
@@ -7128,7 +7133,7 @@
 
 @node ALLOWGETSET, BUTTONACT, SPECIAL VARIABLES, SPECIAL VARIABLES
 @comment  node-name,  next,  previous,  up
-@unnumberedsubsec allowgetset
+@unnumberedsec allowgetset
 @cindex allowgetset
 
 @example
